build.xml

```diff
@@ -604,12 +605,12 @@
     <!-- create custom import configurations to load the MARC filename into the building facet to help
           test cases to limit searches to within specific collections of test records.
           Note: ${srcdir}/local/import/import.properties is created in install.php. -->
-    <copy file="${srcdir}/import/marc_local.properties" tofile="${srcdir}/local/import/marc-${BASENAME}.properties" overwrite="true" />
-    <append destFile="${srcdir}/local/import/marc-${BASENAME}.properties" text="building=&quot;${BASENAME}&quot;${line.separator}" />
+    <copy file="${srcdir}/import/marc_local.properties" tofile="${localdir}/import/marc-${BASENAME}.properties" overwrite="true" />
```

Comment:
Is it also worth replacing the `${srcdir}/local` reference in the comment above this for consistency?

Reply:
Yes, done.